corehq/apps/userreports/pillow_utils.py

```diff
@@ -103,6 +103,10 @@ def rebuild_table(adapter, diffs=None):
         adapter.log_table_rebuild_skipped(source='pillowtop', diffs=diff_dicts)
         return
 
+    if not config.is_static:
+        config.set_build_queued(reset_init_fin=False)
```

Comment:
and we will reset awaiting to False if the table is being migrated and not rebuilt by pillows?

Reply:
Ah! Thank you, this is the feedback I'm looking for.

If the table is migrated and not rebuilt, are these the correct flags? 
```python
    config.meta.build.awaiting = False

    # Reset these, or leave unchanged?
    config.meta.build.initiated = None
    config.meta.build.finished = False  # Or `True`?
```

Reply:
I can confirm that this is correct and is the one that blocks users from taking more action.
`config.meta.build.awaiting = False`

I am not sure how and where others are relevant.

Reply:
0ebcf5dbec9

Reply:
Sorry I missed this earlier,
I believe at this point, `awaiting` is already set to True since that is done in the view so we should not need to queue it again here. The bug was that the flag was not getting removed.